### DISCUSSION TITLE
error.log又刷了

### DISCUSSION THREAD

**Discussion by Miuzarte** [Tsuk1ko/CQ-picfinder-robot#369] [问题求助] [answered]
![1}ILKQEWD}_`22KC2748@C7](https://user-images.githubusercontent.com/66856838/198993815-81780c40-725b-4312-92cb-06ba4db1e492.png)
![WVTW_ 635 )_HL JHJBYA6Y](https://user-images.githubusercontent.com/66856838/198993826-5ebcae85-d019-40f4-a6ae-ccce00f53f13.png)

**Comment by Tsuk1ko** [comment]
因为对方没有动态所以被以为是接口错误（

我去掉这个 log 好了

**Reply by Tsuk1ko** [reply]
去掉了，更新下

**Comment by Miuzarte** [comment, marked as answer]
> 因为对方没有动态所以被以为是接口错误（
> 
> 我去掉这个 log 好了

绷，原来是我又把直播间号输进去了
